<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThermoFunWidget</class>
 <widget class="QMainWindow" name="ThermoFunWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ThermoFun Tabulation of Thermodynamic Properties</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="keyWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="keySplitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
            <property name="spacing">
             <number>1</number>
            </property>
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QTextEdit" name="taskQuery">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="listsWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="lstSplitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="inWidget" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <item row="3" column="2">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Units</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="5">
             <widget class="QSpinBox" name="pPrecision">
              <property name="toolTip">
               <string extracomment="Output precision / digits after decimal point (in Fixed Format)">Number of digits after decimal point for pressure</string>
              </property>
              <property name="toolTipDuration">
               <number>5</number>
              </property>
              <property name="value">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2" alignment="Qt::AlignLeft">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Data type</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2" colspan="4">
             <widget class="QLineEdit" name="pName">
              <property name="toolTip">
               <string>Calculation task name</string>
              </property>
             </widget>
            </item>
            <item row="6" column="4" colspan="2">
             <widget class="QComboBox" name="pSolventSymbol">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string>Symbol of the solvent used for calculating properties of disolved solute species</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_7">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Pressure</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="4">
             <widget class="QLabel" name="label_10">
              <property name="toolTip">
               <string>Number of digits after decimal point</string>
              </property>
              <property name="statusTip">
               <string>Precision of temperature values</string>
              </property>
              <property name="text">
               <string>Precision</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="3">
             <widget class="QComboBox" name="pPunit">
              <item>
               <property name="text">
                <string>b bar</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>B kbar</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>p Pa</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>P MPa</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>A atm</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Units</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QComboBox" name="pTunit">
              <property name="currentText">
               <string>C Celsius</string>
              </property>
              <item>
               <property name="text">
                <string>C Celsius</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>K Kelvin</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="7" column="0" colspan="8">
             <widget class="QLabel" name="label_9">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Properties table</string>
              </property>
             </widget>
            </item>
            <item row="4" column="4">
             <widget class="QLabel" name="label_12">
              <property name="toolTip">
               <string>Number of digits after decimal point</string>
              </property>
              <property name="statusTip">
               <string>Precision of pressure values</string>
              </property>
              <property name="text">
               <string>Precision</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2" alignment="Qt::AlignLeft">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Task Name</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="2" colspan="2">
             <widget class="QComboBox" name="typeBox">
              <property name="toolTip">
               <string>Type of data to calculate properties for</string>
              </property>
             </widget>
            </item>
            <item row="6" column="3">
             <widget class="QLabel" name="label_13">
              <property name="toolTip">
               <string>Symbol of the solvent used for calculating properties of disolved solute species</string>
              </property>
              <property name="text">
               <string>Solvent Symbol</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="6">
             <widget class="QLineEdit" name="pComment">
              <property name="toolTip">
               <string>Calculation task comment</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Temperature</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="5">
             <widget class="QSpinBox" name="tPrecision">
              <property name="toolTip">
               <string extracomment="Output precision / digits after decimal point (in Fixed Format)">Number of digits after decimal point for temperature</string>
              </property>
              <property name="toolTipDuration">
               <number>5</number>
              </property>
              <property name="value">
               <number>2</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="outWidget" native="true">
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <property name="spacing">
             <number>2</number>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Temperature T points</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="calcStatus">
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Status bar, what to do next and error messages</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>872</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew_Reset_Task_Definition"/>
    <addaction name="separator"/>
    <addaction name="actionExport_CfgFile"/>
    <addaction name="actionImport_CfgFile"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionContents"/>
    <addaction name="separator"/>
    <addaction name="actionVersion"/>
    <addaction name="actionAuthors"/>
    <addaction name="actionLicense"/>
    <addaction name="separator"/>
    <addaction name="action_Help_About"/>
   </widget>
   <widget class="QMenu" name="menu_Calc">
    <property name="title">
     <string>&amp;Calculate</string>
    </property>
    <addaction name="actionCalculate_Properties"/>
    <addaction name="actionCalculate_TP"/>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>&amp;Preferences</string>
    </property>
    <addaction name="actionCalculate_Substances_Records_from_Dependent_Reactions"/>
    <addaction name="actionCalculate_Reactions_Records_from_Reactants"/>
    <addaction name="separator"/>
    <addaction name="action_Set_Elemets_to_reactionsets"/>
    <addaction name="actionFixed_output_number_format"/>
   </widget>
   <widget class="QMenu" name="menuSelect">
    <property name="title">
     <string>Se&amp;lect</string>
    </property>
    <addaction name="actionSelect_ThermoDataSet"/>
    <addaction name="actionSelect_Elements"/>
    <addaction name="actionSelect_Substances"/>
    <addaction name="actionSelect_Reactions"/>
   </widget>
   <widget class="QMenu" name="menuSet">
    <property name="title">
     <string>&amp;Set</string>
    </property>
    <addaction name="actionReset_TP"/>
    <addaction name="actionRealloc_TP"/>
    <addaction name="separator"/>
    <addaction name="actionChange_Property_list"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionShow_Results"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuSelect"/>
   <addaction name="menuSet"/>
   <addaction name="menu_Calc"/>
   <addaction name="menuView"/>
   <addaction name="menuPreferences"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="statusTip">
    <string/>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Reset_Task_Definition"/>
   <addaction name="actionExport_CfgFile"/>
   <addaction name="actionImport_CfgFile"/>
  </widget>
  <widget class="QToolBar" name="nameToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelect_ThermoDataSet"/>
   <addaction name="actionReset_TP"/>
   <addaction name="actionChange_Property_list"/>
   <addaction name="actionCalculate_Properties"/>
   <addaction name="actionShow_Results"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Help_About"/>
  </widget>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionImport_CfgFile">
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/ShowFilesIcon24.png</normaloff>:/Icons/ShowFilesIcon24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Task Definition from File ...</string>
   </property>
   <property name="iconText">
    <string>Read Task Definition from File ...</string>
   </property>
   <property name="toolTip">
    <string>Read calculation setup from file ...</string>
   </property>
  </action>
  <action name="actionExport_CfgFile">
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/SaveCurrentRecordIcon24.png</normaloff>:/Icons/SaveCurrentRecordIcon24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save Task Definition to File ...</string>
   </property>
   <property name="iconText">
    <string>Save Task Definition to File ...</string>
   </property>
   <property name="toolTip">
    <string>Save calculation setup to file...</string>
   </property>
  </action>
  <action name="action_Help_About">
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/QuestionMessageIcon24.png</normaloff>:/Icons/QuestionMessageIcon24.png</iconset>
   </property>
   <property name="text">
    <string> Help &amp;About</string>
   </property>
   <property name="toolTip">
    <string>Show Help VIewer ...</string>
   </property>
  </action>
  <action name="actionAuthors">
   <property name="text">
    <string>Au&amp;thors ...</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>&amp;License ...</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="text">
    <string>Help &amp;Contents ...</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>&amp;Version ...</string>
   </property>
  </action>
  <action name="actionCalculate_Properties">
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/CalculateRecordDataIcon24.png</normaloff>:/Icons/CalculateRecordDataIcon24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Properties ...</string>
   </property>
  </action>
  <action name="actionCalculate_TP">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/RTparmModuleIcon24.png</normaloff>:/Icons/RTparmModuleIcon24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Tabulate the temperature and pressure points</string>
   </property>
  </action>
  <action name="actionShow_Results">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/ShowGEMResultsIcon24.png</normaloff>:/Icons/ShowGEMResultsIcon24.png</iconset>
   </property>
   <property name="text">
    <string>Table with &amp;Results  ...</string>
   </property>
  </action>
  <action name="actionChange_Property_list">
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/DocumentsPileIcon24.png</normaloff>:/Icons/DocumentsPileIcon24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Properties to Calculate ...</string>
   </property>
   <property name="toolTip">
    <string>Set properties to calculate</string>
   </property>
  </action>
  <action name="actionRealloc_TP">
   <property name="text">
    <string>&amp;Size of Temperature and Pressure points list ...</string>
   </property>
   <property name="toolTip">
    <string>Realloc TP pairs list</string>
   </property>
  </action>
  <action name="actionReset_TP">
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/SelectTP24.png</normaloff>:/Icons/SelectTP24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Temperature and Pressure points ...</string>
   </property>
   <property name="toolTip">
    <string>Set temperature and pressure points</string>
   </property>
  </action>
  <action name="actionNew_Reset_Task_Definition">
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/NewCreateRecordIcon24.png</normaloff>:/Icons/NewCreateRecordIcon24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New/Reset Task Definition</string>
   </property>
   <property name="toolTip">
    <string>Reset window</string>
   </property>
  </action>
  <action name="actionSelect_ThermoDataSet">
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/ICompModuleIcon24.png</normaloff>:/Icons/ICompModuleIcon24.png</iconset>
   </property>
   <property name="text">
    <string>Select &amp;ThermoDataSet ...</string>
   </property>
   <property name="toolTip">
    <string>Select thermodynamic dataset</string>
   </property>
  </action>
  <action name="action_Set_Elemets_to_reactions">
   <property name="text">
    <string> &amp;Set Elemets to reactions</string>
   </property>
  </action>
  <action name="actionCalculate_Substances_Records_from_Dependent_Reactions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Calculate &amp;Substances Records from Dependent Reactions</string>
   </property>
  </action>
  <action name="actionCalculate_Reactions_Records_from_Reactants">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Calculate &amp;Reactions Records from Reactants</string>
   </property>
  </action>
  <action name="action_Set_Elemets_to_reactionsets">
   <property name="text">
    <string> &amp;Set Elemets to reactionsets</string>
   </property>
   <property name="toolTip">
    <string>Set Elemets to ReactionSets</string>
   </property>
  </action>
  <action name="actionSelect_Elements">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select &amp;Elements ...</string>
   </property>
  </action>
  <action name="actionSelect_Substances">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select &amp;Substances ...</string>
   </property>
  </action>
  <action name="actionSelect_ReactionSets">
   <property name="text">
    <string>Select ReactionSets ...</string>
   </property>
  </action>
  <action name="actionSelect_Reactions">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select Reactions ...</string>
   </property>
  </action>
  <action name="actionFixed_output_number_format">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fixed output number format</string>
   </property>
   <property name="toolTip">
    <string>Output values that have a fixed number of digits after the decima point</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../ThermoFunDemoGUI.qrc"/>
 </resources>
 <connections/>
</ui>
